pilot/pkg/config/kube/gateway/conversion.go

```diff
@@ -292,6 +293,15 @@ func buildHTTPVirtualServices(
 			vs.Route = route
 		}
 
+		if r.Timeouts != nil {
+			if r.Timeouts.Request != nil {
+				vs.Timeout = durationpb.New(r.Timeouts.Request.Duration)
+			}
+			if r.Timeouts.BackendRequest != nil {
+				vs.Retries.PerTryTimeout = durationpb.New(r.Timeouts.BackendRequest.Duration)
```

Comment:
```suggestion
				if vs.Retries == nil {
					vs.Retries = &istio.HTTPRetry{}
				}
				vs.Retries.PerTryTimeout = durationpb.New(r.Timeouts.BackendRequest.Duration)
```

Reply:
addressed in 14443afd7b4a012c35984c3e8242daca517528e7